src/gui/macutilities.mm

```diff
@@ -31,10 +31,10 @@
 #import <Cocoa/Cocoa.h>
 #include <objc/message.h>
 
+#include <QList>
```

Comment:
This file doesn't seem to need `QVector` nor `QList`. Try omitting this header.

Reply:
> This file doesn't seem to need `QVector` nor `QList`. Try omitting this header.

Removed..